src/aspnet/StackExchange.Redis.Extensions.AspNetCore/StackExchange.Redis.Extensions.AspNetCore.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
Should net 6 be here due to framework doing the heavy lifting ie dependency management?

Reply:
My opinion is **no** because .NET 6 will be EOL after .NET 9 releases, though we can keep supporting it for a while.

Let @imperugo decide 😄 .

Reply:
IMHO we can remove .NET 6 and all unsupported framework, but I'm pretty sure there will be people who will complain and want support

Reply:
Yeap Was more thinking about who are yet to upgrade